### PR TITLE
Add underwater spin sound

### PR DESCRIPTION
**Pull request by Koopa1018**
# Description of changes
Adds an underwater variant of the spin sound.

Implementation is based on #69 (obligatory number joke here). The spin sound is now routed through "SFX Clear In Water," which bypasses the underwater low-pass filter for a much clearer sound. The sound is designed with this in mind, and should blend with other underwater sounds perfectly fine.
# Reviewer(s)

@GTcreyon 
@Charpurrr 
@brickblock369 

# Issue(s)
Checks one box on #35.

### CODE REVIEW

File: default_bus_layout.tres [hunk 1]
```diff
@@ -1,41 +1,59 @@
...
+bus/1/name = "~~Water Muffle:"
 bus/1/solo = false
 bus/1/mute = false
 bus/1/bypass_fx = false
 bus/1/volume_db = 0.0
 bus/1/send = "Master"
-bus/2/name = "SFX"
+bus/1/effect/0/effect = SubResource( 2 )
+bus/1/effect/0/enabled = true
+bus/2/name = "~~Water Muffle:"
```
Comment (GTcreyon) [inline default_bus_layout.tres:26]: Why are there two buses named `~~Water Muffle:`?

File: default_bus_layout.tres [hunk 2]
```diff
@@ -1,41 +1,59 @@
...
+bus/4/effect/0/enabled = true
+bus/5/name = "~~SFX Clear In Water~~"
+bus/5/solo = false
+bus/5/mute = false
+bus/5/bypass_fx = false
+bus/5/volume_db = 0.0
+bus/5/send = "SFX"
+bus/5/effect/0/effect = SubResource( 3 )
+bus/5/effect/0/enabled = true
+bus/6/name = "~~SFX Clear In Water~~"
```
Comment (GTcreyon) [inline default_bus_layout.tres:54]: Equally, why are there 2 buses named `~~SFX Clear In Water~~`?

And there is no bus named `~Water Verb:` for bus 6 to send to.

File: classes/player/player.tscn [hunk 3]
```diff
@@ -467,7 +466,7 @@ bus = "SFX"
 [node name="SpinSFX" type="AudioStreamPlayer" parent="."]
 stream = ExtResource( 18 )
 volume_db = -17.0
-bus = "SFX"
+bus = "~~SFX Clear In Water~~"
```
Comment (GTcreyon) [inline classes/player/player.tscn:469]: This should be "~Water Verb:" I think?

**Review by GTcreyon — CHANGES_REQUESTED** [review]
Running the game causes the console to be flooded with the following error:
```
E 0:00:00.935   set_bus_mute: Index p_bus = -1 is out of bounds (buses.size() = 7).
  <C++ Source>  servers/audio_server.cpp:764 @ set_bus_mute()
  <Stack Trace> options_menu.gd:53 @ _process()
```

It seems that default_bus_layout.tres is misconfigured. Could you take a look at that?

**Comment by Koopa1018** [comment]
Ahhh frick, that looks like a failed merge :/
(I didn't think to separate this and #69 until after I'd already made them together in one branch. It seems I failed to separate them properly.)

**Comment by Koopa1018** [comment]
Alright, should be fixed.

**Comment by Koopa1018** [comment]
Easier said than done, due to the way the sound is authored (I'd have to implement the fade in code, then make sure it only happens if the user's not still holding the button).

If it's a cleaner ending you want, it'd be way easier to just play a short secondary sound when the user stops spinning.

**Comment by GTcreyon** [comment]
> If it's a cleaner ending you want, it'd be way easier to just play a short secondary sound when the user stops spinning.

That seems reasonable.

**Comment by Koopa1018** [comment]
Arright, should sound better now.

**Review by GTcreyon — APPROVED** [review]
Minor issue where the sound continues playing if you exit water while spinning, but it's not too significant. This is easiest to pull off by power swimming in t1r2.

**Pull request merged**